lisa/runner.py

```diff
@@ -122,13 +128,18 @@ def _initialize_runners(self) -> None:
         # group filters by runner type
         runner_filters: Dict[str, List[schema.BaseTestCaseFilter]] = {}
         for raw_filter in self._runbook.testcase_raw:
-            runner_type = raw_filter.get(constants.TYPE, constants.TESTCASE_TYPE_LISA)
-            raw_filters: List[schema.BaseTestCaseFilter] = runner_filters.get(
-                runner_type, []
-            )
-            if not raw_filters:
-                runner_filters[runner_type] = raw_filters
-            raw_filters.append(raw_filter)
+            # by default run all cases unless 'enable' is specified as false
+            enabled = raw_filter.get("enable", True)
```

Comment:
Since there are two members used. so it's better to load raw_filter to schema here for validation. 

`filter = schema.BaseTestCaseFilter.schema().load(raw_filter)`

The filter should have type and enable fields.